samples/azure/self-host/Core_7/HostWorker/SafeRoleEnvironment.cs

```diff
@@ -0,0 +1,177 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+
+[DebuggerNonUserCode]
+static class SafeRoleEnvironment
```

Comment:
Should `SafeRoleEnvironment` be a linked file? It's the same for all.

Reply:
Good point I'll do

Reply:
Hmm... There is only one endpoint. Are you suggesting to cross-link between versions? I wouldn't do that